flake8_class_attributes_order/checker.py

```diff
@@ -239,12 +282,30 @@ def _get_ordering_errors(cls, model_parts_info) -> List[Tuple[int, int, str]]:
                 ))
         return errors
 
+    @classmethod
+    def _get_node_weights(cls) -> Mapping[str, int]:
+        if ClassAttributesOrderChecker.class_attributes_order:
+            node_type_weights = cls.FIXED_NODE_TYPE_WEIGHTS.copy()
+            node_to_configured_weight = {
+                k: v for v, k in enumerate(
+                    ClassAttributesOrderChecker.class_attributes_order,
+                    start=len(node_type_weights))
+            }
+
+            for node_type, node_type_path in cls.CONFIGURABLE_NODE_TYPES.items():
+                for node_type_or_supertype in node_type_path:
+                    if node_type_or_supertype in node_to_configured_weight:
+                        node_type_weights[node_type] = node_to_configured_weight[node_type_or_supertype]
+                        break
+
+            return node_type_weights
+        elif ClassAttributesOrderChecker.use_strict_mode:
```

Comment:
Looks like we should fire a warning if both `class_attributes_order` and `use_strict_mode` are specified.